src/ApiCore/TransportFactory.php

```diff
@@ -201,54 +131,25 @@ protected static function createSslChannelCredentials()
         return ChannelCredentials::createSsl();
     }
 
-    /**
-     * @param string|null $transport
-     * @return string
-     */
-    private static function handleTransport($transport)
+    private static function defaultTransport()
     {
-        if ($transport) {
-            return strtolower($transport);
-        }
-
-        $transport = self::getGrpcDependencyStatus()
+        return self::getGrpcDependencyStatus()
             ? 'grpc'
             : 'rest';
-
-        return $transport;
     }
 
     /**
-     * @param array $args
-     * @return AuthWrapper
+     * @param $serviceAddress
+     * @return array
+     * @throws ValidationException
      */
-    private static function buildAuthWrapper(array $args)
+    private static function validateServiceAddress($serviceAddress)
     {
-        $authHttpHandler = $args['authHttpHandler'] ?: HttpHandlerFactory::build();
-
-        if (isset($args['credentialsLoader'])) {
-            $credentialsLoader = $args['credentialsLoader'];
-        } else {
-            self::validateNotNull($args, ['scopes']);
-
-            $credentialsLoader = self::getADCCredentials(
-                $args['scopes'],
-                $authHttpHandler
-            );
-
-            if ($args['enableCaching']) {
-                if (!isset($args['authCache'])) {
-                    $args['authCache'] = new MemoryCacheItemPool();
-                }
-
-                $credentialsLoader = new FetchAuthTokenCache(
-                    $credentialsLoader,
-                    $args['authCacheOptions'],
-                    $args['authCache']
-                );
-            }
+        $components = explode(':', $serviceAddress);
+        if (count($components) !== 2) {
+            throw new ValidationException(
+                'Invalid serviceAddress. Expected format "<address>:<port>", got ' . $serviceAddress);
```

Comment:
The `);` should be on a new line. Also, if a port is not included does it seem reasonable to default to 443? That is what we do currently and I don't see a reason to change it.

Reply:
Makes sense, will support.